aea/cli/utils/package_utils.py

```diff
@@ -467,46 +461,6 @@ def validate_author_name(author: Optional[str] = None) -> str:
     return valid_author
 
 
-def validate_registry_type(default_registry: Optional[str] = None) -> str:
```

Comment:
Since we're providing defaults for the registry flag there is no chance that `default_registry` can be `None` so we don't need to check